lib/core/include/irods/rcMisc.h

```diff
@@ -360,9 +360,15 @@ int getAttriInAttriArray(const char* objPath,
                          int* outDataMode,
                          char** outChksum);
 
+// clang-format off
+__attribute__((deprecated("trimSpaces is deprecated. Use boost's trim instead.")))
```

Comment:
We can drop the message since the compiler will state that information for us. There's also no need to speak of boost's trim function since the person may be working in a C-only environment.

Reply:
Removed message.